polaris.shopify.com/src/components/Header/Header.tsx

```diff
@@ -0,0 +1,100 @@
+import { useState, useEffect } from "react";
+import Link from "next/link";
+import Image from "next/image";
+
+import GlobalSearch from "../GlobalSearch";
+import MaxPageWidthDiv from "../MaxPageWidthDiv";
+import Button from "../Button";
+import SideMenu from "../SideMenu";
+import NavItems from "../NavItems";
+
+import shopifyLogo from "../../../public/shopify-logo.svg";
+import hamburguerIcon from "../../../public/images/icon-hamburguer.svg";
+import styles from "./Header.module.scss";
+
+interface Props {
+  currentSection?: string;
+}
+
+function Header({ currentSection }: Props) {
+  const [showMenu, setShowMenu] = useState(false);
+
+  useEffect(() => {
+    // it ensures that the menu is closed after the screen is resized
```

Comment:
Shopify has taught me this pattern which I like: Always try to avoid comments and let the code do the talking instead. Can you remove this comment and instead clarify the code? Perhaps renaming `handleResize` to `setNavVisiblityOnResize` would be just as clear?